shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/LocalSecrets.java

```diff
@@ -33,26 +30,17 @@ public String getKey(String secretName) throws SecretRetrievalException {
         String key = "";
 
         try {
-            key = readSecretFromFileSystem(secretName);
-        } catch (SecretRetrievalException exception) {
-            logger.logWarning(
-                    "Not finding the " + secretName + " on the filesystem, searching in resources");
             key = readSecretFromResources(secretName);
+        } catch (SecretRetrievalException exception) {
+            logger.logWarning("Not finding the " + secretName + " in resources");
+            throw new SecretRetrievalException(
+                    "Could not find " + secretName + "in resources", exception);
```

Comment:
No-need to rewrap the `SecretRetrievalException` that comes out of the `readSecretFromResources` method.  You can completely nuke the try/catch stuff and just call the `readSecretFromResources` method directly.